eth/ethconfig/config.go

```diff
@@ -36,6 +36,14 @@ import (
 	"github.com/ethereum/go-ethereum/params"
 )
 
+const (
+	// ChainDbName is the name of the chain database
+	ChainDbName = "chaindata"
```

Comment:
maybe `DatabaseName` and `DatabaseNamespace`?

Reply:
sounds reasonable :)